packages/ui-extensions/src/surfaces/admin/components/patterns/examples/calloutCard.html

```diff
@@ -0,0 +1,36 @@
+<s-section id="callout-section">
+  <s-grid gridTemplateColumns="1fr auto" gap="small-400" alignItems="start">
+    <s-grid
+      gridTemplateColumns="@container (inline-size <= 480px) 1fr, auto auto"
+      gap="base"
+      alignItems="center"
+    >
+      <s-grid gap="small-200">
+        <s-heading>Ready to create your custom puzzle?</s-heading>
+        <s-paragraph>
+          Start by uploading an image to your gallery or choose from one of our templates.
+        </s-paragraph>
+        <s-stack direction="inline" gap="small-200" wrap="nowrap">
+          <s-button onClick="console.log('clicked')"> Upload image </s-button>
+          <s-button tone="neutral" variant="tertiary"> Browse templates </s-button>
+        </s-stack>
+      </s-grid>
+      <s-stack alignItems="center">
```

Comment:
Does the `s-stack` do anything here since there's only one item in its children?